### ISSUE TITLE
wrong DockerHub's image name used for Cygnus-LD  (`cygnus-ngsi-ld` instead of  `cygnus-ld`)

### ISSUE THREAD

**Issue by andrii482** [telefonicaid/fiware-cygnus#2217]
`cygnus-ngsi-ld` - is name of docker image that  is  using in this repository now.

`cygnus-ld` - is name of  docker image  that hosting in reality  on  `hub.docker.com`

https://github.com/telefonicaid/fiware-cygnus/blob/bdbbd58054d2ecb82e1718bf92f2646b6b665b5d/doc/cygnus-ngsi-ld/installation_and_administration_guide/install_with_docker.md?plain=1#L67

https://github.com/telefonicaid/fiware-cygnus/blob/bdbbd58054d2ecb82e1718bf92f2646b6b665b5d/README.md?plain=1#L63

https://hub.docker.com/r/fiware/cygnus-ld/

https://hub.docker.com/r/fiware/cygnus-ngsi-ld/

**Comment by fgalan** [comment]
Maybe the contributor to NGSI-LD can provide information on this. I'm CCing it.

CC: @anmunoz

**Comment by anmunoz** [comment]
> Maybe the contributor to NGSI-LD can provide information on this. I'm CCing it.
> 
> CC: @anmunoz

Hi, 

I am checking the repository and I notice that the docker hub for cygnus have been migrated by the FIWARE organisation. Thus, the official repository is https://hub.docker.com/r/fiware/cygnus-ld/ . Please @jason-fox can you confirm that? and in that case, please @fgalan Could you update the documentation?

BR,

**Comment by fgalan** [comment]
> Could you update the documentation?

As Cygnus-LD mantainer I think it would be better if you do it. Could you do the PR with the update, please? Thanks!

**Comment by jason-fox** [comment]
> can you confirm that?

Yes. The fiware docker image for Cygnus NGSI-LD is called https://hub.docker.com/r/fiware/cygnus-ld
The fiware docker image for Cygnus NGSI-v2  is called https://hub.docker.com/r/fiware/cygnus-ngsi
